CWE/events/East Turkestan Additions.txt

```diff
@@ -0,0 +1,1404 @@
+
+		NOT = { XIJ = { war_with = ETS } #Ili Rebellion }
+
+	}
+
+	mean_time_to_happen = { months = 3 }
+
+	option = {
+		name = "Restore the local currency and combat inflation."
+		money = 1000
+		any_pop = {
+			ideology = { factor = 0.10 value = socialist }
+			militancy = -3
+			consciousness = -3
+		}
+		ai_chance = { 
+			factor = 0.8
+		}
+	}
+
+	option = {
+		name = "We can't risk upsetting China."
+		add_country_modifier = { name = political_strife duration = 395 }
+		add_country_modifier = { name = economic_malaise duration = 1825 }
+		any_pop = {
+			limit = { OR = { culture = chinese culture = hui }
+			ideology = { factor = 0.10 value = nationalist }
+			militancy = 3
+			consciousness = 3
+		}
+		ai_chance = { 
+			factor = 0.2
+		}
+	}
+}
+
+
+#Mao demands our formal surrender
+country_event = {
+  id = 8023814
+  title = "Incorporation of Xinjiang"
+  desc = EVTDESC8023814
+  picture = "nwo2_east_turkestan"
+  
+	fire_only_once = yes
+
+	trigger = {
+		tag = CHI
+		year = 1949 
+		NOT = { year = 1990 }
+		
+		OR = { government = proletarian_dictatorship government = proletarian_dictatorship1 }
+		is_vassal = no 
+		
+		XIJ = { 
+			exists = yes 
+			is_vassal = no 
+			
+			#Xinjiang must border the PRC
+			neighbour = THIS
+		}
+		
+		NOT = { XIJ = { war_with = ETS } #Ili Rebellion }
+
+		#China won the Civil War
+		has_global_flag = CHI_won_civil_war
+	}
+
+	mean_time_to_happen = { months = 1 }
+
+	option = {
+		name = "Demand their surrender."
+		1600 = {
+			change_province_name = "Urumqi"
+		}
+		XIJ = { country_event = 8023815 } 
+		ai_chance = { factor = 0.9 }
+	}
+
+	option = {
+		name = "Leave them be"
+		prestige_factor = -0.05
+		any_pop = {
+			militancy = 3
+			consciousness = 3
+		}
+		relation = { who = XIJ value = 200 }
+		
+		ai_chance = { factor = 0.1 }
+	}
+}
+
+#Mao demands our formal surrender - Response
+country_event = {
+	id = 8023815
+	title = "A Chinese Communist Victory"
+	desc = EVTDESC8023815
+	picture = "xij_mao"
+
+	is_triggered_only = yes
+
+	option = {
+		name = "Bloodshed must be avoided, we will negotiate a surrender of Xinjiang. (Game Over)"
+		CHI = {
+			inherit = XIJ
+		}
+		ai_chance = { 
+			factor = 0.8
+		}
+	}
+
+	option = {
+		name = "We must fight to preserve the KMT's rule in Xinjiang!"
+		CHI = {
+			war = {
+				target = XIJ
+				attacker_goal = { casus_belli = annex_core_country }
+				defender_goal = { casus_belli = status_quo }
+				call_ally = no
+			}
+		}
+		ai_chance = { 
+			factor = 0.2
+		}
+	}
+}
+
+
+#Chiang demands submission
+country_event = {
+  id = 8023816
+  title = "Integration of the Frontier"
+  desc = EVTDESC8023816
+  picture = "nwo2_east_turkestan"
+  
+	fire_only_once = yes
+
+	trigger = {
+		tag = KMT
+		year = 1949 
+		NOT = { year = 1990 }
+		
+		is_vassal = no 
+		
+		XIJ = { 
+			exists = yes 
+			is_vassal = no 
+			
+			#Xinjiang must border the KMT
+			neighbour = THIS
+		}
+		
+		NOT = { XIJ = { war_with = ETS } #Ili Rebellion }

```

Comment:
NOT = { XIJ = { war_with = ETS } } #Ili Rebellion

Reply:
Okay. all concerns should now be addressed.